test.js

```diff
@@ -127,14 +143,21 @@ test('the arguments object can be passed in, spread', t => {
   t.true(foo('far', {}));
 });
 
-test('allows buffer in the required config', t => {
-  const buffer = new Buffer('robin');
+test('Function.prototype.apply with the arguments object', t => {
+  function foo () {
+    return argsert.apply(this, arguments);
```

Comment:
@bcoe so this is probably the way that https://github.com/yargs/yargs/pull/773 would make use of this lib :)